<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1279</width>
    <height>845</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="Sidebar" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>791</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>1111</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:white;</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPushButton" name="SwitchToHome">
         <property name="styleSheet">
          <string notr="true">border:none</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../../../../2144.png</normaloff>../../../../../2144.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="SwitchToNutritions">
         <property name="styleSheet">
          <string notr="true">border:none</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../../../../nutritions_icon-removebg-preview.png</normaloff>../../../../../nutritions_icon-removebg-preview.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>5060</width>
           <height>60</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="SwitchToWorkout">
         <property name="styleSheet">
          <string notr="true">border:none</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../../../../training_icon-removebg-preview.png</normaloff>../../../../../training_icon-removebg-preview.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>80</width>
           <height>80</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="settingsbutton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">border:none;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../../../../settings icon.png</normaloff>../../../../../settings icon.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:black</string>
      </property>
      <property name="currentIndex">
       <number>5</number>
      </property>
      <widget class="QWidget" name="Training_Page">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" name="widget" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="spacing">
            <number>120</number>
           </property>
           <property name="leftMargin">
            <number>100</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>100</number>
           </property>
           <item>
            <widget class="QPushButton" name="gymmodebutton">
             <property name="styleSheet">
              <string notr="true">font:40pt;
font-family:&quot;Bebas Neue&quot;;
border-radius:15px;
background-color:white;</string>
             </property>
             <property name="text">
              <string>GYM</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="calisthenicsmodebutton">
             <property name="styleSheet">
              <string notr="true">font:40pt;
font-family:&quot;Bebas Neue&quot;;
border-radius:15px;
background-color:white;</string>
             </property>
             <property name="text">
              <string>calisthenics</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Home_Page">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" name="Header" native="true">
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="Title">
             <property name="styleSheet">
              <string notr="true">font-size: 90px;
font-weight: bold;
color: #ffffff;
qproperty-alignment: AlignCenter;
font-family:&quot;Bebas Neue&quot;;</string>
             </property>
             <property name="text">
              <string>Welcome Back,</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="username_show">
             <property name="styleSheet">
              <string notr="true">font-size: 35px;
font-weight: bold;
color: #ffffff;
qproperty-alignment: AlignCenter;</string>
             </property>
             <property name="text">
              <string>Username</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="quotes">
             <property name="styleSheet">
              <string notr="true">font-size: 20px;
font-weight: bold;
color: #ffffff;
qproperty-alignment: AlignCenter;</string>
             </property>
             <property name="text">
              <string>Quotes</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="NPCs" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QWidget" name="Workout" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>30</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="decorpic">
                <property name="text">
                 <string/>
                </property>
                <property name="pixmap">
                 <pixmap>decor4.png</pixmap>
                </property>
                <property name="scaledContents">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="decor">
                <property name="styleSheet">
                 <string notr="true">color:white;
font:7pt;
</string>
                </property>
                <property name="text">
                 <string>Are you ready for the grind ? Catch me up !</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="Nutritions" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="0,0">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="decor_2">
                <property name="styleSheet">
                 <string notr="true">color:white;
font:7pt;
</string>
                </property>
                <property name="text">
                 <string>Follow your diet, follow your grind</string>
                </property>
                <property name="textFormat">
                 <enum>Qt::AutoText</enum>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="decorpic_2">
                <property name="text">
                 <string/>
                </property>
                <property name="pixmap">
                 <pixmap>../../../../../image-removebg-preview (1).png</pixmap>
                </property>
                <property name="scaledContents">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Nutritions_Page">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QScrollArea" name="scrollArea_3">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_3">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>1175</width>
             <height>770</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QWidget" name="widget_2" native="true">
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <property name="spacing">
                <number>10</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
                <widget class="QPushButton" name="movetochatbot">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">font:40pt;
font-family:&quot;Bebas Neue&quot;;
border-radius:15px;
background-color:white;</string>
                 </property>
                 <property name="text">
                  <string>Open AI Coach</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="terms_disclaimer">
                 <property name="styleSheet">
                  <string notr="true">color:white;font-size:13pt;font-family:Bebas Neue;font-weight:bold;text-decoration: underline;</string>
                 </property>
                 <property name="text">
                  <string>AI coach terms and disclaimer</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Gym_Page">
       <widget class="QScrollArea" name="scrollArea">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>1196</width>
          <height>794</height>
         </rect>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>1194</width>
           <height>792</height>
          </rect>
         </property>
         <widget class="QWidget" name="GridWidget" native="true">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>1196</width>
            <height>2000</height>
           </rect>
          </property>
         </widget>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="CalisPage">
       <widget class="QScrollArea" name="scrollArea_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>1196</width>
          <height>794</height>
         </rect>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>1194</width>
           <height>792</height>
          </rect>
         </property>
         <widget class="QWidget" name="GridWidget_2" native="true">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>1196</width>
            <height>2000</height>
           </rect>
          </property>
         </widget>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="Terms_Disclaimer">
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="Terms_Title">
          <property name="styleSheet">
           <string notr="true">color:white;font-size:30pt;font-family:Bebas Neue;font-weight:bold;text-decoration: underline;</string>
          </property>
          <property name="text">
           <string>AI COACH TERMS AND DISCLAIMER</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="plainTextEdit">
          <property name="styleSheet">
           <string notr="true">color:white;font-size:15pt;font-family:Bebas Neue;font-weight:bold;</string>
          </property>
          <property name="plainText">
           <string>1. **Purpose of AI CoACH**
   The aI COACH is designed to help users with fitness, workouts, nutrition, and general guidance. It is an assistant, not a medical or professional advisor.

2. **User Responsibility**

   * Users must use the AI Coach responsibly and understand that information provided is for educational purposes.
   * Users are responsible for their own actions when following workout or nutrition advice.

3. **Accuracy of Information**

   * While we aim to provide helpful and accurate guidance, the chatbot may not always be perfect.
   * The AI COACH may simplify explanations and may not cover every possible scenario.

4. **Prohibited Use**

   * Do not use the AI Coach for illegal, harmful, or abusive purposes.
   * Do not try to hack, manipulate, or overload the AI COACH system.

5. **Privacy and Data**

   * Chat messages may be stored temporarily to improve the chatbot experience.
   * Do not share sensitive personal information.

6. **No Guarantees**

   * The AI COACH does not guarantee results, success, or outcomes from following advice.
   * Users should consult professionals for serious health, fitness, or medical concerns.

7. **Changes to Terms**

   * We may update these terms from time to time. Users are encouraged to review them regularly.

**By using this AI COACH, you agree to follow these terms and use it responsibly.**
</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1279</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
